gogoproto/gogo.proto

```diff
@@ -28,7 +28,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-syntax = "proto2";
+syntax = "proto3";
```

Comment:
Was it necessary to make this change? I think we could use the original files here.

Reply:
I don't remember changing this. Probably not needed.

Reply:
Alright, so let's use the original file.